riscv/src/lib.rs

```diff
@@ -56,6 +58,13 @@ impl RuntimeLibs {
             ..self
         }
     }
+
+    pub fn with_splt_vec(self) -> Self {
```

Comment:
I think we can add this always to the main machine

Reply:
Is enabled with poseidon2 now.